props/blaster.h

```diff
@@ -273,8 +273,10 @@ class Blaster : public PROP_INHERIT_PREFIX PropBase {
     }
   }
 
+#ifndef PROPS_DUAL_PROP_H
```

Comment:
I wonder if this might be a better fix:

void DoMotion(const Vec3& motion, bool clear) override {
  PropBase::DoMotion(Vec3(), clear);
}

(No #ifdefs required.)

Reply:
Tested, that works.

Reply:
> 

Wait - hold that thought. That breaks Clash to Unjam.

Reply:
Doing `PropBase::DoMotion(motion, clear);` instead seems to be right. Commit made.